BambooTracker/BambooTracker.pro

```diff
@@ -42,55 +42,89 @@ CONFIG += c++14
 
 # C/C++ compiler flags
 msvc {
-  CPPFLAGS += /Wall /Wp64 /WX
-  CPPFLAGS += /source-charset:utf-8
+  message("Qt configured with MSVC")
+  message("Compiler is version" $$QT_MSC_FULL_VER)
+  CPP_WARNING_FLAGS += /Wall /Wp64 /WX
+  CPP_WARNING_FLAGS += /source-charset:utf-8
 }
 else:if(gcc|clang) {
```

Comment:
This `gcc|clang` is technically meaningless. `clang` is handled as a derivate of and additionally defines `gcc` so `gcc` would be enough. (Same technically applies for Intel's `icc` compiler -`intel_icc` option - but we don't test for it and I have no way to test it anyway)

I think it's better to keep this in some form for a quicker overview of what compilers we expect and support though. maybe `clang:!intel_icc:gcc` would be better in that we explicitly exclude `icc`?

Reply:
I think it's better to keep it for clarity.

> `clang|if(gcc:!intel_icc)`

I don't know how many users are using icc (it's new to me), but I agree with that, in that we should explicitly indicate that we know that it and gcc are different.